src/algorithms/tracking/CKFTracking.cc

```diff
@@ -7,6 +7,8 @@
 #include <Acts/Definitions/TrackParametrization.hpp>
 #include <Acts/Definitions/Units.hpp>
 #include <Acts/EventData/GenericBoundTrackParameters.hpp>
+#include <Acts/EventData/TrackStateProxy.hpp>
```

Comment:
Fails to compile for older versions due to missing this class. v32 has it. Previously in `Core/include/Acts/EventData/MultiTrajectory.hpp`. See https://github.com/acts-project/acts/commit/01f0877bc691ec45a393fb50fe667499146dac58.